practice-two/src/style/components/input.css

```diff
@@ -0,0 +1,10 @@
+.input-placeholder {
+  padding-left: 10px;
```

Comment:
You can add padding-left: 10px in `cta__input` class instead of create a new class here